types/dom-chromium-installation-events/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for non-npm package dom-chromium-installation-events 101.0
+// Project: https://developer.mozilla.org/en-US/docs/Web/API/BeforeInstallPromptEvent
+// Definitions by: Sergey Kozlov <https://github.com/dartess>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type AppBannerPromptOutcome = 'accepted' | 'dismissed';
```

Comment:
this is global script, so you can write this eg. that way, without exports:

```ts
type AppBannerPromptOutcome = 'accepted' | 'dismissed';

interface PromptResponseObject {
    readonly outcome: AppBannerPromptOutcome;
    readonly platform: string;
}

interface BeforeInstallPromptEvent extends Event {
    readonly platforms: string[];
    readonly userChoice: Promise<PromptResponseObject>;
    prompt(): Promise<void>;
}

declare var BeforeInstallPromptEvent: {
    prototype: BeforeInstallPromptEvent;
    new (type: string, eventInitDict: EventInit): BeforeInstallPromptEvent;
};

interface WindowEventMap {
    beforeinstallprompt: BeforeInstallPromptEvent;
    appinstalled: Event;
}

interface Window {
    onappinstalled?: ((this: Window, ev: Event) => any) | null;
    onbeforeinstallprompt?: ((this: Window, ev: BeforeInstallPromptEvent) => any) | null;
}
```

Reply:
thanks, applied